cmd/pcap/index/command.go

```diff
@@ -65,8 +66,16 @@ func (c *Command) Run(args []string) error {
 	if err != nil {
 		return err
 	}
+	warn := make(chan string)
 	defer f.Close()
```

Comment:
Nit: Don't separate the defer from the constructor it services.
```suggestion
	defer f.Close()
	warn := make(chan string)
```